frontend/src/views/RoutesView.vue

```diff
@@ -24,15 +24,19 @@
       <div :class="['collapse', { show: routeVisible === route.id }]">
         <div class="card-body">
           <p>
-            <span class="font-weight-bold">{{ $t('routes.description') }}</span>
+            <span class="font-weight-bold">{{ $t('routes.description') }} </span>
             <span>{{ route.description }}</span>
           </p>
           <p>
-            <span class="font-weight-bold">{{ $t('routes.starting_mileage') }}</span>
+            <span class="font-weight-bold">{{ $t('routes.car') }} </span>
+            <span>{{ route.car.plates }}</span>
+          </p>
+          <p>
+            <span class="font-weight-bold">{{ $t('routes.starting_mileage') }} </span>
             <span>{{ route.startMileage }}</span>
           </p>
           <p>
-            <span class="font-weight-bold">{{ $t('routes.ending_mileage') }}</span>
+            <span class="font-weight-bold">{{ $t('routes.ending_mileage') }} </span>
```

Comment:
why the extra space? there is no space after span tag..

Reply:
IMHO it looks a little bit better with space between first and second span. 
![zaznaczenie_001](https://user-images.githubusercontent.com/28116755/48891906-fb844c00-ee3c-11e8-89e2-01a5107373f6.png)